src/interfaces/ListInterface.ts

```diff
@@ -0,0 +1,8 @@
+interface ListInterface {
+  id?: number;
```

Comment:
id seems like the least likely thing to be null, right?  i mean, assuming it's being stoped in some kind of database

Reply:
I'd bet it's because when you create one, it doesn't have the id. There's probably a ton of ways to handle this, but I've seen `id=-1` and then when it hits the db, a real id would be populated. Or `id=null` and the same thing (in this case `id: number | null`). Or just different types for each case.